layouts/index.redirects

```diff
@@ -10,4 +10,7 @@
 {{ $lang := .File.ContentBaseName -}}
 /docs/{{ $lang }}   /docs/instrumentation/{{ $lang }}
 /docs/{{ $lang }}/*  /docs/instrumentation/{{ $lang }}/:splat
-{{ end -}}
\ No newline at end of file
+{{ end -}}
+
+/docs/reference/specification/common/common  /docs/reference/specification/common
+/docs/reference/specification/common/common/* /docs/reference/specification/common/:splat
```

Comment:
Ideally these lines should be in the front-matter of the page files in question.

Reply:
- #1372